tests/test_calc_resid.py

```diff
@@ -0,0 +1,37 @@
+"""
+This file tests the calculate residual helper function, used in .fit().
+"""
+import sys
+import numpy as np
+
+sys.path.insert(1, '../')
+import sigmet
```

Comment:
I think `from sigmet import Sigmet` solves the problem copied from [here](https://stackoverflow.com/questions/18069716/why-am-i-getting-module-object-is-not-callable-in-python-3). Not relevant to test formatting